app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayout_bottom_nav"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".DashboardFragment" >
+
+    <LinearLayout
+        android:id="@+id/frame_layout"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/size_10x"
+        android:orientation="horizontal"
+        android:weightSum="4"
+        android:background="@color/n_0"
+        android:layout_alignParentBottom="true">
+
+        <LinearLayout
```

Comment:
Hmm did you tried https://developer.android.com/reference/com/google/android/material/bottomnavigation/BottomNavigationView?
Implementation details:
https://m2.material.io/components/bottom-navigation/android#using-bottom-navigation
or
https://github.com/material-components/material-components-android/blob/master/docs/components/BottomAppBar.md